backend/inspirehep/records/api/data.py

```diff
@@ -24,6 +25,7 @@ def create(
         **kwargs,
     ):
         record = super().create(data, **kwargs)
+        record.update_authors_records_table()
```

Comment:
I am a bit confused with this one. Why we need to update the authors one here ?

Reply:
good question, because we want to display data records in author profiles